types/model/name.go

```diff
@@ -243,7 +243,7 @@ func (n Name) Filepath() string {
 		panic("illegal attempt to get filepath of invalid name")
 	}
 	return strings.ToLower(filepath.Join(
-		n.Host,
+		strings.Replace(n.Host, ":", "%", 1),
```

Comment:
Do we want to do this only on windows? OR we may need a migration path for non windows users.

Reply:
Probably will need a migration path. Switching back and forth will make things complicated. I recommend we stick with one way.

Reply:
only doing this for windows risk the same problem as `sha256:` on ntfs filesystem. this should probably be for all operating systems

Reply:
fwiw the number of users affected would be low since most will be using the default host

Reply:
Updated with migration. PTAL.